examples/TestAsync.purs

```diff
@@ -0,0 +1,48 @@
+module Main where
+
+import Prelude
+import Data.Either
+import Data.Maybe
+import Control.Monad.Eff
+import Control.Monad.Eff.Console(log,CONSOLE())
+import Control.Monad.Eff.Exception
+
+import Node.FS
+import qualified Node.FS.Async as A
+import qualified Node.Path as FP
+import qualified Node.Buffer as B 
+
+-- exercise the file descriptor based async IO functions
+
+main :: forall eff . Eff (fs::FS,err::EXCEPTION,console::CONSOLE|eff) Unit
+main = do
+  let path1 = FP.concat( ["examples", "TestAsync.purs"] )
+      path2 = FP.concat( ["examples", "TestAsync.purs.partial"] )
+      buf = B.create 1000
+  A.fdOpen path1 R Nothing $ \v -> case v of
+    (Left err) -> log ("err:" ++ show err)
```

Comment:
This was just meant to be a trivial demo to meet your request "Nothing fancy - I'm just thinking something like calling them once and checking nothing obviously wrong happens".

Are there plans to build a test suite for this library? The existing example file doesn't typecheck with the current compiler/libraries.